robottelo/hosts.py

```diff
@@ -1275,9 +1275,7 @@ def create_custom_environment(self, repo='generic_1'):
             f'--target-dir /etc/puppetlabs/code/environments/{env_name}/modules/'
         )
         smart_proxy = (
-            entities.SmartProxy()
```

Comment:
There are other references in the module still, if you want to include updating those here.